### ISSUE TITLE
Crash when plugging into external monitors

### ISSUE THREAD

**Issue by scottopell** [rickbutton/workspacer#44]
I opened my laptop, workspacer started re-arranging windows, then I plugged in my external monitors.

All of my windows started appearing on one of my two external monitors, then the following exception occurred:

```
System.InvalidOperationException: Sequence contains no matching element
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source, Func`2 predicate)
   at workspacer.NativeMonitorContainer.GetMonitorAtPoint(Int32 x, Int32 y)
   at workspacer.WorkspaceManager.SwitchFocusedMonitorToMouseLocation()
   at workspacer.KeybindManager.DoMouseEvent(MouseEvent evt)
   at workspacer.KeybindManager.MouseHook(Int32 nCode, UIntPtr wParam, IntPtr lParam)
```

**Comment by scottopell** [comment]
Crash was on 0.9.5.91

**Comment by rickbutton** [comment]
from the stack, it looks like the crash occurred during a mouse click callback, so it was likely clicked before workspacer managed to detect the new display settings (new monitors), so when you clicked inside of a new monitor workspacer didn't know about, it broke. this is fixed in `0.9.5.93`
